buildpack.md

```diff
@@ -118,11 +119,12 @@ The lifecycle MAY return an error to the platform if two or more buildpacks with
 
 Executable: `/bin/detect <platform[AR]> <plan[E]>`, Working Dir: `<app[AR]>`
 
-| Input             | Description
-|-------------------|----------------------------------------------
-| `$0`              | Absolute path of `/bin/detect` executable
-| `<platform>/env/` | User-provided environment variables for build
-| `<platform>/#`    | Platform-specific extensions
+| Input                        | Description
+|------------------------------|----------------------------------------------
+| `$0`                         | Absolute path of `/bin/detect` executable
+| `<platform>/env/`            | User-provided environment variables for build
+| `<platform>/#`               | Platform-specific extensions
+| `$CNB_ASSETS/<asset-sha256>` | Any assets provided by the platform to a buildpack. Both the `$CNB_ASSETS` environment variable and any specific asset are optional.
```

Comment:
Hmm... the implementation specifically stripped this var from the detect env: https://github.com/buildpacks/lifecycle/blob/771ac0b9d0d56cd3baaf28355498a0a636ea6182/acceptance/detector_test.go#L139

Which is preferred?

Reply:
I think that while we only mentioned the build phase explicitly in the RFC we didn't intentionally exclude detect and instead merely forget it. Providing this env var during detect seems like the easiest and most consistent thing to do.